main.py

```diff
@@ -80,39 +80,37 @@ def main(argv: Sequence[str]) -> int:
     port = None
 
     if (
-        app.utils.is_valid_inet_address(app.settings.SERVER_ADDR)
-        and app.settings.SERVER_PORT is not None
+        app.utils.is_valid_inet_address(app.settings.APP_HOST)
+        and app.settings.APP_PORT is not None
     ):
-        host = app.settings.SERVER_ADDR
-        port = app.settings.SERVER_PORT
+        host = app.settings.APP_HOST
+        port = app.settings.APP_PORT
     elif (
-        app.utils.is_valid_unix_address(app.settings.SERVER_ADDR)
-        and app.settings.SERVER_PORT is None
+        app.utils.is_valid_unix_address(app.settings.APP_HOST)
+        and app.settings.APP_PORT is None
     ):
-        uds = app.settings.SERVER_ADDR
+        uds = app.settings.APP_HOST
```

Comment:
if we're forcing the use of docker then we can't use unix sockets unless they manually bridge the file... shouldn't we just remove this?